bagel/tests/data/README.md

```diff
@@ -15,18 +15,10 @@ Example inputs to the CLI
 | 8            | valid, based on ex2 has multiple participant_id columns                                                                                                                  | valid, based on ex2 multiple participant_id column annotations                       | fail*              |
 | 9            | invalid, based on example 6 but contains an unannotated value for `group`                                                                                                | valid, based on example 6                                                            | fail               |
 | 10           | valid, same as example 6                                                                                                                                                 | valid, based on example 6 but contains extra `"MissingValues"` not found in the .tsv | pass, with warning |
-| synthetic    | valid, has `participant` and `session` IDs corresponding to the [`synthetic` example BIDS dataset](https://github.com/bids-standard/bids-examples/tree/master/synthetic) | valid                                                                                | pass               |
 | 11           | invalid, ex 6 with missing entries in `participant_id` and `session_id` columns                                                                                          | valid, based on example 6                                                            | fail               |
 | 12           | Valid, same as example 2                                                                                                                                                 | Valid, based on example 2 but missing BIDS "Levels" attribute for group column       | Pass, with warning |
 | 13           | Valid, same as example_synthetic                                                                                                                                         | Valid, based on example_synthetic but with mismatched levels for group column        | Pass, with warning |
 | 14           | Valid, same as example 2                                                                                                                                                 | Valid, based on example 2, but with an extra column annotation without Neurobagel    | Pass               |
 | 15           | Valid, same as example 2                                                                                                                                                 | Invalid, based on example 2, but participant ID column lacks Neurobagel annotations | Fail                   |
 
-`* this is expected to fail until we enable multiple participant_ID handling`.
-
-## Example expected CLI outputs
```

Comment:
Can we add a line here linking to the `neurobagel_examples` repo for our reference examples for valid CLI inputs + outputs? Or in the main CLI README, if you think it's more appropriate there.